pkg/k8s/init.go

```diff
@@ -196,6 +196,10 @@ func GetNodeSpec() error {
 		if option.Config.K8sRequireIPv4PodCIDR || option.Config.K8sRequireIPv6PodCIDR {
 			return fmt.Errorf("node name must be specified via environment variable '%s' to retrieve Kubernetes PodCIDR range", k8sConst.EnvNodeNameSpec)
 		}
+		if option.Config.KubeProxyReplacement != option.KubeProxyReplacementDisabled {
+			log.Info("K8s node name is empty. BPF NodePort might not be able to auto detect " +
+				"devices (if not specified)")
```

Comment:
Can we check `devices` to see if it's specified to clarify this info message further?